module/Company/src/Company/Model/CompanyPacket.php

```diff
@@ -3,16 +3,14 @@
 namespace Company\Model;
 
 use Doctrine\ORM\Mapping as ORM;
-//use Doctrine\Common\Collections\ArrayCollection;
-//use Zend\Permissions\Acl\Role\RoleInterface;
-//use Zend\Permissions\Acl\Resource\ResourceInterface;
+use Doctrine\Common\Collections\ArrayCollection as ArrayCollection;
 
 /**
  * CompanyPacket model.
  *
  * @ORM\Entity
  */
-class CompanyPacket //implements RoleInterface, ResourceInterface
+class CompanyPacket
```

Comment:
Since GEWIS calls what we sell to companies *packages*, I definitely have a different opinion here.